### PR TITLE
Gpl 320 upgrade to rails 6

### PR DESCRIPTION
**Pull request by stevieing**
There is some stuff that I have had to do that might need some explanation so happy to do a group code review if you both feel that would be beneficial.
Based on what I have had to do to get this to work that it would be beneficial to add a new story to persist the configuration.

### CODE REVIEW

File: bin/rails [hunk 1]
```diff
@@ -1,11 +1,4 @@
 #!/usr/bin/env ruby
-# frozen_string_literal: true
-
-begin
-  load File.expand_path('spring', __dir__)
-rescue LoadError => e
-  raise unless e.message.include?('spring')
-end
```
Comment (KatyTaylor) [inline bin/rails]: What's the reason for removing these bits, and should you also remove bin/spring ?

File: config/cable.yml [hunk 2]
```diff
@@ -2,9 +2,9 @@ development:
   adapter: async
 
 test:
-  adapter: async
+  adapter: test
```
Comment (KatyTaylor) [inline config/cable.yml]: what's the reason for this change?

File: config/cable.yml [hunk 3]
```diff
@@ -2,9 +2,9 @@ development:
...
 
 production:
   adapter: redis
   url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
-  channel_prefix: traction-service_production
+  channel_prefix: traction_service_production
```
Comment (KatyTaylor) [inline config/cable.yml]: also, what will this change affect?

File: config/initializers/cors.rb [hunk 4]
```diff
@@ -1,18 +1,17 @@
...
-end
+# Rails.application.config.middleware.insert_before 0, Rack::Cors do
+#   allow do
+#     origins 'example.com'
+#
+#     resource '*',
+#       headers: :any,
+#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
+#   end
+# end
```
Comment (KatyTaylor) [inline config/initializers/cors.rb]: How is CORS handled now?

File: spec/models/concerns/material_spec.rb [hunk 5]
```diff
@@ -1,5 +0,0 @@
-require 'rails_helper'
-
-shared_examples "material" do
-  it { is_expected.to have_one(:tube) }
-end
```
Comment (KatyTaylor) [inline spec/models/concerns/material_spec.rb]: why get rid of this test?

File: bin/rails (hunk 1 above)
Comment (stevieing) [reply bin/rails]: Part of the Rails upgrade. A number of files are replaced automatically during the upgrade so these were over written.

File: config/cable.yml (hunk 2 above)
Comment (stevieing) [reply config/cable.yml]: As above.

File: config/cable.yml (hunk 3 above)
Comment (stevieing) [reply config/cable.yml]: As above. We don't use action cable.

File: config/initializers/cors.rb (hunk 4 above)
Comment (stevieing) [reply config/initializers/cors.rb]: As above. We handle cors in application.rb. We could move it to initializer but not a problem.

File: spec/models/concerns/material_spec.rb (hunk 5 above)
Comment (stevieing) [reply spec/models/concerns/material_spec.rb]: This is a shared example and is a duplicate of spec/support/material.rb. I was getting a redefinition warning.
